applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_base_solver.py

```diff
@@ -98,6 +98,18 @@ def __init__(self, model, custom_settings):
             self.main_model_part.ProcessInfo.SetValue(KratosMultiphysics.DOMAIN_SIZE, domain_size)
             self.solver_imports_model_part = True
 
+        element_name = self.settings["element_replace_settings"]["element_name"].GetString()
+        if (self.settings["use_orthogonal_subscales"].GetBool() is True):
+            if element_name in ("QSConvectionDiffusionExplicit","DConvectionDiffusionExplicit"):
+                self.main_model_part.ProcessInfo.SetValue(KratosMultiphysics.OSS_SWITCH, 1)
+            else:
+                KratosMultiphysics.Logger.PrintWarning("::[ConvectionDiffusionBaseSolver]:: ", " W-A-R-N-I-N-G: The selected element", element_name, "does not support OSS projection. Select QSConvectionDiffusionExplicit or DConvectionDiffusionExplicit instead.")
```

Comment:
If you don't switch off the OSS this must be an error.

Reply:
Ok, now this is an error.